src/bokeh/core/enums.py

```diff
@@ -257,11 +257,11 @@ def enumeration(*values: Any, case_sensitive: bool = True, quote: bool = False)
 Align = enumeration(AlignType)
 
 #: Horizontal alignment of a child item
-HAlignType = Literal["top", "center", "bottom"]
+HAlignType = Literal["left", "center", "right"]
 HAlign = enumeration(HAlignType)
 
 #: Vertical alignment of a child item
-VAlignType = Literal["left", "center", "right"]
+VAlignType = Literal["top", "center", "bottom"]
```

Comment:
Have we simply not been using these types at all in the past?

Reply:
Apparently not, but it's a fairly obscure feature and most of the time it's easier to use `Anchor` variant instead.